vsintegration/src/FSharp.Editor/CodeFixes/RenameUnusedValue.fs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System
+open System.Composition
+open System.Threading.Tasks
+open System.Collections.Immutable
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Text
+open Microsoft.CodeAnalysis.CodeFixes
+
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Syntax
+
+open CancellableTasks
+
+module UnusedCodeFixHelper =
+    let getUnusedSymbol textSpan (document: Document) codeFixName =
```

Comment:
Is this used ouside of this file/codefix currently?
If yes, then "RQA" should be used, and module should be internal
If not, then module should be private and autoopened.

Reply:
No that's not used anywhere else, done.

Actually this file should be split into three - keeping it this way here so that git handles it but will do it in a followup. And will add a bunch of testing related to backticks.